testkit/tests/change_configuration.rs

```diff
@@ -66,23 +66,13 @@ fn test_configuration_and_rollbacks() {
     let old_config = testkit.actual_configuration();
     let new_config = proposal.stored_configuration().clone();
 
-    testkit.commit_configuration_change(proposal);
+    testkit.checkpoint();
 
-    testkit.create_blocks_until(Height(10));
-    assert_eq!(testkit.actual_configuration(), new_config);
-    testkit.rollback(2);
-    assert_eq!(testkit.actual_configuration(), old_config);
+    testkit.commit_configuration_change(proposal);
```

Comment:
It makes sense to create another checkpoint after `commit_configuration_change` and test that its rollback does not influence accepting the new configuration.

Reply:
Can you please elaborate on it? We don't really have arguments to properly fuzz test and I find it unnecessary for this situation.